Documentation/cookbook_overview.rst

```diff
@@ -1,10 +1,12 @@
 Overview
 ========
 
-This documentation is a collection of useful dynamic simulation examples to help users build models from the scratch with little efforts. Each example is demonstrated carefully with geometry building, parameter setup and result visualization. Users are suggested to repeat each example in order to get a comprehensive idea of how to set up dynamic simulation models with SeisSol. The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol) or Readdocs (https://seissol.readthedocs.io/en/latest).
+This documentation is a collection of useful dynamic simulation examples to help users build models from the scratch with little efforts. Each example is demonstrated carefully with geometry building, parameter setup and result visualization. Users are suggested to repeat each example in order to get a comprehensive idea of how to set up dynamic simulation models with SeisSol. The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol).
```

Comment:
Could you (not for now but for later) put each sentence in a new line? This makes the differences between versions much easier to read. (We use the same policy also for writing papers collaboratively.)

Reply:
from scratch
little effort
We expect that these examples may help new users to learn how to set up dynamic simulation models with SeisSol.
I would remove 'The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol).' or maybe you can explain what you mean by that